src/refiners/training_utils/trainer.py

```diff
@@ -141,6 +160,10 @@ def __init__(self, config: ConfigType) -> None:
         self._models: ModelRegistry = {}
         self._callbacks: CallbackRegistry = {}
         self.config = config
+        self.batch_time_m = AverageMeter()
```

Comment:
nit: What does `m` stand for? I prefer to use explicit names (even if verbose).

Reply:
ah meter. Yeah let me fix that